src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -12698,12 +12713,18 @@ void VRFTerminalUnitEquipment::ControlVRF_FluidTCtrl(EnergyPlusData &state,
                 if (this->isSetPointControlled) {
                     if (state.dataLoopNodes->Node(this->coolCoilAirOutNode).Temp > this->coilTempSetPoint) return;
                 } else {
+                    if (QZnReq >= 0.0 && FullOutput >= 0.0) PartLoadRatio = 0.0;
                     return;
                 }
             } else {
                 PartLoadRatio = 0.0;
                 return;
             }
+        } else {
+            if (QZnReq == 0.0 && (FullOutput < 0.0 && NoCompOutput < FullOutput)) {
+                PartLoadRatio = 0.0;
+                return;
+            }
```

Comment:
The PLR is reset to zero when there is no zoneload and the NoLoad component output value is larger than the FullOutput. This change was required to eliminate the overshoot of cooling set-points.